graph/encoding/dot/decode.go

```diff
@@ -41,6 +42,17 @@ type PortSetter interface {
 // Unmarshal parses the Graphviz DOT-encoded data and stores the result in dst.
 // If the number of graphs encoded in data is not one, an error is returned and
 // dst will hold the first graph in data.
+//
+// Attributes that are not IDs are quoted during marshalling, to conform with
```

Comment:
From the spec, this is not restricted to just attributes. We should probably handle all the cases properly.

Reply:
Doing this also for IDs would resolve the issue noted in https://github.com/gonum/gonum/pull/793#issuecomment-453873502

Reply:
Done.